chains/evm/contracts/pools/FastTransferTokenPoolAbstract.sol

```diff
@@ -309,7 +309,7 @@ abstract contract FastTransferTokenPoolAbstract is TokenPool, CCIPReceiver, ITyp
   // @inheritdoc CCIPReceiver
   function _ccipReceive(
     Client.Any2EVMMessage memory message
-  ) internal virtual override onlyRouter {
```

Comment:
If you're making changes like these please provide context. This now looks like major security issue.